app/src/ui/lib/draggable.tsx

```diff
@@ -147,25 +151,25 @@ export class Draggable extends React.Component<IDraggableProps> {
     document.removeEventListener('mousemove', this.onMouseMove)
     mouseScroller.clearScrollTimer()
     this.props.onRemoveDragElement()
-    this.props.onDragEnd(this.isLastElemBelowDropTarget())
-    dragAndDropManager.dragEnded()
+    if (this.props.onDragEnd !== undefined) {
+      this.props.onDragEnd(this.isLastElemBelowDropTarget())
+    }
+    dragAndDropManager.dragEnded(this.isLastElemBelowDropTarget())
   }
 
   /**
    * Compares the last element that the mouse was over during a drag with the
    * css selectors provided in dropTargetSelectors to determine if the drag
    * ended on target or not.
    */
-  private isLastElemBelowDropTarget = (): boolean => {
+  private isLastElemBelowDropTarget = (): DropTargetSelector | undefined => {
```

Comment:
Given this function does not return a `boolean` anymore, maybe it's worth renaming it?
```suggestion
  private lastElemBelowDropTarget = (): DropTargetSelector | undefined => {
```